leaflet/index.d.ts

```diff
@@ -1524,6 +1524,26 @@ declare namespace L {
         export const svg: boolean;
     }
 
+    export namespace Util {
+        export function extend(dest: {}, src?: {}): {};
+        export function create(proto: {}, properties?: {}): {};
+        export function bind(fn: () => void, ...obj: Array<{}>): () => void;
+        export function stamp(obj: {}): number;
+        export function throttle(fn: () => void, time: number, context: {}): () => void;
+        export function wrapNum(num: number, range: number[], includeMax?: boolean): number;
+        export function falseFn(): () => void;
```

Comment:
According to the docs, this returns a function that always returns `false`, so the signature for the return type can be `() => boolean` or maybe even `() => false`